lib/grape/middleware/stack.rb

```diff
@@ -6,14 +6,27 @@ module Middleware
     # It allows to insert and insert after
     class Stack
       class Middleware
-        attr_reader :args, :block, :klass
+        use_ruby2_keywords = Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7')
```

Comment:
Since we only use this once, let's avoid an unnecessary variable and just do 

```ruby
if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7')
...
else
...
end
```

Reply:
I didn't want to do the comparison/computation on every call. Happy to add a memoized static method instead.

Reply:
Right, so put it around method definition.

```ruby
if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.7')
  def use_in
  end
else
  def use_in
  end
end
```

Reply:
If you do want to leave it as a method and be memoized, put it into a separate module, e.g. `Grape::Ruby::Version` ala https://github.com/dblock/with-version (we don't want to take a dependency I think for such seldom usage).